source/openapi-spec.yml

```diff
@@ -26,13 +20,10 @@ paths:
                 properties:
                   data:
                     $ref: "#/components/schemas/Application"
-
-        '401':
+        "401":
           $ref: "#/components/responses/Unauthorized"
-
-        '404':
+        "404":
```

Comment:
Suggestion: if we're writing lots of `.yml`, https://prettier.io is an excellent editor plugin / CLI tool to handle stuff like this automatically ✨ (also does JS, CSS, etc)

Reply:
Yup, agreed! I use it locally but not sure about the whole dev team, a thing we can follow up with.